workspaces/src/types.rs

```diff
@@ -55,3 +59,37 @@ impl InMemorySigner {
         &self.0
     }
 }
+
+// type taken from near_primitives::hash::CryptoHash.
+/// CryptoHash is type for storing the hash of a specific block.
+pub struct CryptoHash(pub [u8; 32]);
```

Comment:
why use newtype for this over just an alias of `[u8; 32]` for simplicity?

Reply:
I just took whatever was in `near_primitives`. It's a newtype there, so stuck with that. I don't know if changing it to an alias of `[u8; 32]` would make it harder to transition later when `near_primitives` is `1.0`

Reply:
why would it be hard to transition? Just wrapping the bytes passed in with `CryptoHash` internally? Am I missing something with what you mean here?

Reply:
I'm just thinking whether or not if we end up just exposing `near_primitives::hash::CryptoHash` later and its still just a newtype which would make it a breaking change to transition over to. But maybe not a huge concern if we force everyone to just rely on our `CryptoHash` from workspaces

Reply:
yeah, the primitives type prob shouldn't be tied to our API. I'm not sure if I'm misunderstanding what you're saying here though. Why would we ever need to expose the primitives type to be a part of our API?

Reply:
I was just thinking of this case, because there was a conversation we had about potentially exposing the `near_primitives`/`near_account_id` types once they released 1.0. But regardless of that, I think the newtype here would be better suited since it might be harder for users to create `CryptoHash`es if it was just an alias of `[u8; 32]`. The `FromStr` impl should allow us to create it easily